playbooks/roles/os_temps/tasks/add_rh_cert.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Add RH IT Cert
+# This is required to pull the OpenShift 3.11 Jenkins image from
+# docker-registry.upshift.redhat.com.  Remove this task after
+# upgrading to Jenkins OCP4 (CVP-2930)
+
+# get kubeadmin password
+- name: "Get kubeadmin password"
+  set_fact:
+    kubeadmin_password: "{{ lookup('file', '~/.crc/machines/crc/kubeadmin-password') }}"
+
+# login as admin to the cluster
+- name: "Login as admin to the Openshift cluster"
+  command: "{{ oc_bin }} login https://api.crc.testing:6443 --username=kubeadmin --password={{ kubeadmin_password }} --insecure-skip-tls-verify=true"
+
+# Add Red Hat CA certs to OpenShift
+- name: "Add Red Hat CA certs to OpenShift"
+  command: "{{ oc_bin }} patch cm registry-certs -n openshift-config --patch='{\"data\":{\"docker-registry.upshift.redhat.com\":\"-----BEGIN CERTIFICATE-----\\nMIIENDCCAxygAwIBAgIJANunI0D662cnMA0GCSqGSIb3DQEBCwUAMIGlMQswCQYD\\nVQQGEwJVUzEXMBUGA1UECAwOTm9ydGggQ2Fyb2xpbmExEDAOBgNVBAcMB1JhbGVp\\nZ2gxFjAUBgNVBAoMDVJlZCBIYXQsIEluYy4xEzARBgNVBAsMClJlZCBIYXQgSVQx\\nGzAZBgNVBAMMElJlZCBIYXQgSVQgUm9vdCBDQTEhMB8GCSqGSIb3DQEJARYSaW5m\\nb3NlY0ByZWRoYXQuY29tMCAXDTE1MDcwNjE3MzgxMVoYDzIwNTUwNjI2MTczODEx\\nWjCBpTELMAkGA1UEBhMCVVMxFzAVBgNVBAgMDk5vcnRoIENhcm9saW5hMRAwDgYD\\nVQQHDAdSYWxlaWdoMRYwFAYDVQQKDA1SZWQgSGF0LCBJbmMuMRMwEQYDVQQLDApS\\nZWQgSGF0IElUMRswGQYDVQQDDBJSZWQgSGF0IElUIFJvb3QgQ0ExITAfBgkqhkiG\\n9w0BCQEWEmluZm9zZWNAcmVkaGF0LmNvbTCCASIwDQYJKoZIhvcNAQEBBQADggEP\\nADCCAQoCggEBALQt9OJQh6GC5LT1g80qNh0u50BQ4sZ/yZ8aETxt+5lnPVX6MHKz\\nbfwI6nO1aMG6j9bSw+6UUyPBHP796+FT/pTS+K0wsDV7c9XvHoxJBJJU38cdLkI2\\nc/i7lDqTfTcfLL2nyUBd2fQDk1B0fxrskhGIIZ3ifP1Ps4ltTkv8hRSob3VtNqSo\\nGxkKfvD2PKjTPxDPWYyruy9irLZioMffi3i/gCut0ZWtAyO3MVH5qWF/enKwgPES\\nX9po+TdCvRB/RUObBaM761EcrLSM1GqHNueSfqnho3AjLQ6dBnPWlo638Zm1VebK\\nBELyhkLWMSFkKwDmne0jQ02Y4g075vCKvCsCAwEAAaNjMGEwHQYDVR0OBBYEFH7R\\n4yC+UehIIPeuL8Zqw3PzbgcZMB8GA1UdIwQYMBaAFH7R4yC+UehIIPeuL8Zqw3Pz\\nbgcZMA8GA1UdEwEB/wQFMAMBAf8wDgYDVR0PAQH/BAQDAgGGMA0GCSqGSIb3DQEB\\nCwUAA4IBAQBDNvD2Vm9sA5A9AlOJR8+en5Xz9hXcxJB5phxcZQ8jFoG04Vshvd0e\\nLEnUrMcfFgIZ4njMKTQCM4ZFUPAieyLx4f52HuDopp3e5JyIMfW+KFcNIpKwCsak\\noSoKtIUOsUJK7qBVZxcrIyeQV2qcYOeZhtS5wBqIwOAhFwlCET7Ze58QHmS48slj\\nS9K0JAcps2xdnGu0fkzhSQxY8GPQNFTlr6rYld5+ID/hHeS76gq0YG3q6RLWRkHf\\n4eTkRjivAlExrFzKcljC4axKQlnOvVAzz+Gm32U0xPBF4ByePVxCJUHw1TsyTmel\\nRxNEp7yHoXcwn+fXna+t5JWh1gxUZty3\\n-----END CERTIFICATE-----\\n\"}}'"
```

Comment:
Adding redhat certificate to upstream is not secure. 
Should consider other options